### PR TITLE
260 proofreading changes for c1

### LINKED ISSUE #260: Proofreading changes for c1

### PR DESCRIPTION
**Pull request by Robinlovelace**
- **First tweaks to c1**
- **Fix quotes**
- **Fixes to section 1.1**
- **D -> -dimensional**
- **Shorten and simplify prose**
- **Fixes to 1.2**
- **Update text, high-level**
- **Quote fixes: " -> single**
- **The the and other fixes**
- **Complete proofing c1, close #260**

### CODE REVIEW

File: 02-attribute-operations.qmd [hunk 1]
```diff
@@ -299,7 +299,7 @@ In the following code example, given the `world_agg3` continent summary (@fig-sp
 
 -   drop the geometry column,
 -   calculate population density of each continent,
--   arrange continents by the number countries they contain, and
+-   arrange continents by the number countries each contains, and
```
Comment (michaeldorman) [inline 02-attribute-operations.qmd:302]: the number *of* countries, right? @Robinlovelace

File: 08-mapping.qmd [hunk 2]
```diff
@@ -493,7 +493,7 @@ For example, here is a plot similar to @fig-faceted-map2, but specifying each pa
 
 ```{python}
 #| label: fig-faceted-map3
-#| fig-cap: 2D layout in a faceted map, using "manual" specification of the panels 
+#| fig-cap: Two-dimensional layout in a faceted map, using "manual" specification of the panels
```
Comment (michaeldorman) [inline 08-mapping.qmd:496]: Right, I just made the same change but now saw you already did it, thanks!

**Review by michaeldorman — APPROVED** [review]
Excellent, thanks @Robinlovelace !

**Comment by Robinlovelace** [comment]
All seems OK now, will merge

**Pull request merged**